cypress/integration/pages/photoGalleryPage/testsForAMPOnly.js

```diff
@@ -0,0 +1,37 @@
+import config from '../../../support/config/services';
+// import envConfig from '../../../support/config/envs';
+// import appConfig from '../../../../src/server/utilities/serviceConfigs';
```

Comment:
Dowe want to keep this commented out code?

Reply:
We can get rid of it, I just commented it out just in case we needed it for writing future tests. Not sure what the standard procedure is for that but happy to remove them.

Reply:
If we have a convention of leaving it around for future tests no worries, just checking it wasn't accidental.